ecosystem/typescript/sdk/examples/typescript/bcs_transaction.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-console */
+import { AptosClient, AptosAccount, FaucetClient, BCS, TxnBuilderTypes } from 'aptos';
+
+const NODE_URL = process.env.APTOS_NODE_URL || 'https://fullnode.devnet.aptoslabs.com';
+const FAUCET_URL = process.env.APTOS_FAUCET_URL || 'https://faucet.devnet.aptoslabs.com';
```

Comment:
I wonder do we need both typescript and javascript examples?  They're pretty close and interchangeable right?